internal/suites/example/compose/authelia/Dockerfile.backend

```diff
@@ -6,7 +6,7 @@ ARG GROUP_ID
 # gcc and musl-dev are required for building go-sqlite3
 RUN addgroup --gid ${GROUP_ID} dev && \
     adduser --uid ${USER_ID} -G dev -D dev && \
-    apk --no-cache add gcc musl-dev
+    apk --no-cache add gcc musl-dev git
```

Comment:
This is needed because reflex does not work anymore without it apparently.

Reply:
Interesting and I was going to ask why this was in here 😁.

This likely would have also been causing issues for #610 right?